haproxy.cfg

```diff
@@ -32,29 +32,22 @@ frontend http_front
     acl host_equitytool req.hdr(host) -i -m beg equitytool.austinmobility.io
     redirect scheme https code 301 if host_equitytool
 
-    # PostgREST Path ACLs
-    acl path_knack_services path_beg /knack-services
-    acl path_legacy_scripts path_beg /legacy-scripts
-    acl path_ctr_data_lake path_beg /ctr-data-lake
-    acl path_parking path_beg /parking
-    acl path_road_conditions path_beg /road-conditions
-    acl path_bond_reporting path_beg /bond-reporting
-
-    # Host and source ACLs
+    # Only go to postgrest if the path is valid AND:
+    #   - either source is 10.0.0.0/8
+    #   - or source is 162.89.100.0/24 and host header is present
+
+    acl path_postgrest path_beg /knack-services /legacy-scripts /ctr-data-lake /parking /road-conditions /bond-reporting
     acl host_postgrest hdr(host) -i atd-postgrest.austinmobility.io
     acl src_postgrest_internal src 10.0.0.0/8
     acl src_postgrest_coa_network src 162.89.100.0/24
 
-    # Only go to postgrest if the path is valid AND:
-    #   - either source is 10.0.0.0/8
-    #   - or source is 162.89.100.0/24 and host header is present
-    use_backend postgrest if (path_knack_services or path_legacy_scripts or path_ctr_data_lake or path_parking or path_road_conditions or path_bond_reporting) and ( src_postgrest_internal or ( src_postgrest_coa_network and host_postgrest ) )
+    use_backend postgrest if path_postgrest src_postgrest_internal
+    use_backend postgrest if path_postgrest src_postgrest_coa_network host_postgrest
```

Comment:
I feel like this is probably a rule of thumb that I've learned here: write multiple simple `use_backends` instead of one mondo, complex one.

Reply:
cool, so this is routing to the postgrest backend only if it matches one of the `path_postgrest` paths and if it is coming from the COA network or the internal network

it is definitely easier for me to understand now with the `path_postgrest` paths grouped together 👍